src/pages/UserList/ContextMenu/ContextMenu.js

```diff
@@ -127,13 +128,13 @@ const ContextMenu = ({ user, anchorRef, refetchUsers, onClose }) => {
                                 dense
                             />
                         )}
-                        {access.update && twoFactorEnabled && (
+                        {canReset2FA && (
                             <MenuItem
                                 label={i18n.t(
-                                    'Disable Two Factor Authentication'
+                                    'Reset two factor authentication'
```

Comment:
We are calling this "Reset two factor authentication" not "Disable" (this is Morten's suggestion)